src/components/HeapAvatars/HeapAvatars.js

```diff
@@ -1,94 +1,87 @@
 import React, {useCallback, useState} from 'react';
 import {findUserById} from '../../appFunctions';
-import {AreaAvatar} from '../AreaAvatar/AreaAvatar';
-import {PlaceholderAvatar} from '../PlaceholderAvatar/PlaceholderAvatar';
-import {ListUserSearch} from '../ListUserSearch/ListUserSearch';
+import AreaAvatar from '../AreaAvatar/AreaAvatar';
+import PlaceholderAvatar from '../PlaceholderAvatar/PlaceholderAvatar';
+import ListUserSearch from '../ListUserSearch/ListUserSearch';
 
 import styles from './HeapAvatars.less';
 
-export function HeapAvatars(props) {
-    let {changeable = true, maxShowPosition = 10, owners, usersInfo, onChangeHeap} = props;
+export default function HeapAvatars(props) {
+    let {mutable = true, maxShowPosition = 10, owners, usersData, onChangeHeap} = props;
 
-    let numPosition = owners.length + changeable;                                         
+    let numPosition = owners.length + mutable;                                         
```

Comment:
I don't get it
you've just raped JS, if `mutable` is boolean why do you use math with it?

Reply:
i need to add 1 depending on muttable value. Why can't I apply the implicit conversion (boolean => int)?

let numPosition = owners.length + (mutable ? 1 : 0);
Is it more correct?

Reply:
actually you can leave coercion, may be bad naming but I'm ok if you feel ok about it :)